schema/partner_artist.js

```diff
@@ -4,33 +4,36 @@ import Artist from "./artist/index"
 import numeral from "./fields/numeral"
 import { IDFields } from "./object_identification"
 import { GraphQLString, GraphQLObjectType, GraphQLNonNull, GraphQLBoolean } from "graphql"
+import { connectionDefinitions } from "graphql-relay"
 
-export const PartnerArtistType = new GraphQLObjectType({
```

Comment:
I flattened out some of the fields here b/c I needed to access them in the `connectionDefinitions` below. I probably could have somehow kept them as-is and still been able to access/use them, but I couldn't figure it out, so I just started refactoring/flattening out. The end result is identical, but I'm able to just use `fields` below and it worked as expected.